### ISSUE TITLE
Can Save as GIST

### ISSUE THREAD

**Issue by sasankaweera123** [itsNue/my_notes#1]
If you have your notes you can put those as gist (University/Working On)

**Comment by itsNue** [comment]
yup, since I'm using obsidian to take notes locally it's quite easy to maintain a repo for that.
